src/FGFDMExec.h

```diff
@@ -84,7 +84,7 @@ CLASS DOCUMENTATION
     executive is subsequently directed to load the chosen aircraft specification
     file:
 
-    @code{.cpp}
+    @code
```

Comment:
Please don't remove the `{.cpp}` elements
```suggestion
    @code{.cpp}
```

Reply:
Ops ... Sorry, in fact I noticed it after I had committed the correction and I immediately correct it.